examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -216,9 +250,16 @@ class EnergyEvseDelegate : public EnergyEvse::Delegate
 
     /* PREF attributes */
     DataModel::Nullable<uint32_t> GetNextChargeStartTime() override;
+    CHIP_ERROR SetNextChargeStartTime(DataModel::Nullable<uint32_t> newValue);
```

Comment:
instead of `newValue` could we name it in some way to specify measurement?

I think this may be `epochSec` for example.

Also below: required energy is `energyMilliWattH`.

TargetSoC seems to be better because type is encoded as perced.

Reply:
Parameters renamed.